persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/Events.scala

```diff
@@ -1,144 +0,0 @@
-package com.evolutiongaming.akkaeffect.persistence
-
-import cats.data.{NonEmptyList => Nel}
-import cats.syntax.all._
-import cats.kernel.{Eq, Semigroup}
-import cats.{Apply, Eval, NonEmptyTraverse, Order, Show}
-
-/**
-  *
-  * @param values inner Nel[A] will be persisted atomically,
-  *               This applies some restrictions on persistence layer.
-  *               Please do not overuse this feature
-  * @tparam A event
-  */
-final case class Events[+A](values: Nel[Nel[A]]) { self =>
```

Comment:
sad that it does not recognise that it is just moved

Reply:
well, yes, git not so smart. Thus full filed will be marked as created by me without any previous history